src/controllers/addpluginpage.js

```diff
@@ -1,155 +1,239 @@
-define(["jQuery", "loading", "libraryMenu", "globalize", "connectionManager", "emby-button"], function($, loading, libraryMenu, globalize, connectionManager) {
-    "use strict";
+define(['jQuery', 'loading', 'libraryMenu', 'globalize', 'connectionManager', 'emby-button'], function ($, loading, libraryMenu, globalize, connectionManager) {
+    'use strict';
 
     function populateHistory(packageInfo, page) {
-        var html = "";
-        var length = Math.min(packageInfo.versions.length, 10);
-        for (var i = 0; i < length; i++) {
+
```

Comment:
There's a lot of unnecessary newlines.

Reply:
https://github.com/jellyfin/jellyfin-web/pull/395/commits/c1c9d2d5055bacfc26ece03943fb21830153b444